pyoomph/c_output.py

```diff
@@ -85,6 +92,41 @@ def emit_instruction(self, ins: ir.Instruction) -> str:
             return f"{self.emit_var(ins.var)} = {ins.value};\n"
 
         if isinstance(ins, ir.VarCpy):
+            if (
+                isinstance(ins.dest, ir.LocalVariable)
+                and not isinstance(ins.source, ir.LocalVariable)
+                and isinstance(ins.dest.type, FunctionType)
+            ):
+                # Convert C function into a struct representing Oomph function
+
+                functype = ins.dest.type
+                source = self.emit_var(ins.source)
+                dest = self.emit_var(ins.dest)
+
+                # TODO: this is in the wrong place
```

Comment:
I think adding stuff to `function_defs` is fine, because we are adding a function definition, even though it's `static`:

```suggestion
```